### PR TITLE
Update using-test-with-regexp.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Update the question to have lower case `false` and `true` and make it a two answer question

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/javascript-core/javascript-strings-ii/using-test-with-regexp.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/javascript/javascript-core/javascript-strings-ii/using-test-with-regexp.md?ref=3f89d5734d044568005f757ab4c861c16ec054b2)

File: javascript/javascript-core/javascript-strings-ii/using-test-with-regexp.md [hunk 1]
```diff
@@ -45,22 +45,22 @@ containsAtSchool.test(str); // false
 
 ## Practice
 
-What does the following code snippet return? ???
+What does the following code snippet return?
```
Comment (kapnobatai137) [inline javascript/javascript-core/javascript-strings-ii/using-test-with-regexp.md:48]: what do you think about this?

```suggestion
What will the `test`s return?
```

**Review by kapnobatai137 — APPROVED** [review]

**Review by kapnobatai137 — APPROVED** [review]
🚀

**Pull request merged**